components/questions/TagsSelect.tsx

```diff
@@ -109,6 +119,84 @@ export function TagsSelect({
             // 2px box shadow
```

Comment:
We could improve this by using the `classNames` prop instead of `styles`, so we can use tailwind styles and keep it consistent with the rest of our styles. Notably could use `max-sm` instead of doing manual media breakpoints. Probably not worth switching costs unless Claude can ~one-shot it! https://react-select.com/styles#the-classnames-prop